rosdep/python.yaml

```diff
@@ -2811,6 +2811,39 @@ python-tk:
     wily_python3: [python3-tk]
     xenial: [python-tk]
     xenial_python3: [python3-tk]
+python-torch-cuda75-pip:
+  debian:
+    pip:
+      packages: ['http://download.pytorch.org/whl/cu75/torch-0.1.11.post4-cp27-none-linux_x86_64.whl']
```

Comment:
If you ask apt to install a package it will install the latest version available or upgrade that package if it's already installed. This is slightly different in that it will recursively update packages. But that's because you can't simply do a `pip upgrade` like you can do an `apt-get upgrade`. The problem is that rosdep should not be invoking the install rule if the package is already installed, because it should detect that it's installed an doesn't need to do so.